code-examples/php/usage-examples/find.php

```diff
@@ -0,0 +1,23 @@
+<?php
+require_once __DIR__ . "/vendor/autoload.php";
```

Comment:
Be consistent about whitespace between code blocks across multiple files.

Reply:
It would be worth using phpcs to format the code in the same way every time, in an automated way.